app/scripts/modules/core/pipeline/config/triggers/docker/dockerTrigger.module.js

```diff
@@ -60,28 +73,41 @@ module.exports = angular.module('spinnaker.core.pipeline.trigger.docker', [
     function initializeImages() {
       loadImages().then(function (images) {
         $scope.images = images;
-        $scope.registryMap = images.reduce((map, image) => {
-          let key = image.registry;
+        $scope.accountMap = images.reduce((map, image) => {
+          let key = image.account;
           if (!key) {
             return map;
           }
           let all = map[key] || [];
+          let parts = image.repository.split('/');
+          let repository = parts.pop();
+          let org = parts.join('/');
+          if (all.indexOf(org) < 0) {
+            map[key] = all.concat(org);
+          }
+          return map;
+        }, {});
+        $scope.accounts = Object.keys($scope.accountMap);
+        $scope.organizationMap = images.reduce((map, image) => {
+          let parts = image.repository.split('/');
+          let repository = parts.pop();
+          let key = `${image.account}/${parts.join('/')}`;
+          let all = map[key] || [];
           if (all.indexOf(image.repository) < 0) {
             map[key] = all.concat(image.repository);
           }
           return map;
         }, {});
-        $scope.registries = Object.keys($scope.registryMap);
-        updateRepositoryList();
```

Comment:
didn't see reason to call update repository and then update tags if update repository calls update tags, moved it all to updateOrganizationsList() in line 110 of new commit